editor/effects.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+import { parse, stringify } from 'querystring';
+
+/**
+ * WordPress dependencies
+ */
+import { getBlockSettings, switchToBlockType } from 'blocks';
+
+/**
+ * Internal dependencies
+ */
+import { focusBlock, replaceBlocks } from './actions';
+
+export default {
+	REQUEST_POST_UPDATE( action, store ) {
+		const { dispatch } = store;
+		const { postId, edits } = action;
+		const isNew = ! postId;
+		const toSend = postId ? { id: postId, ...edits } : edits;
+
+		new wp.api.models.Post( toSend ).save().done( ( newPost ) => {
+			dispatch( {
+				type: 'REQUEST_POST_UPDATE_SUCCESS',
+				post: newPost,
+			} );
+		} ).fail( ( err ) => {
+			dispatch( {
+				type: 'REQUEST_POST_UPDATE_FAILURE',
+				error: get( err, 'responseJSON', {
```

Comment:
Minor:  I think it would be nicer to extract both, `REQUEST_POST_UPDATE_SUCCESS` and `REQUEST_POST_UPDATE_FAILURE` into an action creator, obviously not a high priority and maybe can be done if/when these actions are triggered elsewhere, to ensure consistency.